mlflow/tracking/fluent.py

```diff
@@ -91,6 +91,9 @@ def set_experiment(
     name via `experiment_name` or by ID via `experiment_id`. The experiment name and ID cannot
     both be specified.
 
+    .. note:: If the experiment being set by name does not exist, a new experiment will be created with the
+        given name. After the experiment has been created, it will be set as the active experiment.
```

Comment:
```suggestion
    .. note:: If the experiment being set by name does not exist, a new experiment will be 
        created with the given name. After the experiment has been created, it will be set 
        as the active experiment.
```